### ISSUE TITLE
Research possible a11y scan requests

### ISSUE THREAD

**Issue by gbinal** [GSA/site-scanning#690]
- Low contrast text
- Missing alt text
- Empty links
- Missing labels
- Empty buttons
- Missing language

First though, we need to make sure that these are requirements for an actual stakeholder outside of IDEA implementation.

**Comment by gbinal** [comment]
New list: screenshot

Questions: 
* Which of the proposed scans are covered by what we currently do?
* How does that list overlap with Cassie's list?  
* What is the value of the other, further scans?  LOE? 
* What are w3c conformance testing tools?  standardization rules?  Are we in sync with that?  
* Do less, but do it really well.

**Comment by gbinal** [comment]
we are currently scanning for: 

```
 'WCAG2AA.Principle1.Guideline1_1',
        HTMLCS_ERROR_CODE,
      );
      a11yHtmlAttributeIssues = getIssueTotalByCategory(
        htmlcsResults,
        'WCAG2AA.Principle4.Guideline4_1',
        HTMLCS_ERROR_CODE,
      );
      a11yColorContrastIssues = getIssueTotalByCategory(
        htmlcsResults,
        'WCAG2AA.Principle1.Guideline1_4',
        HTMLCS_ERROR_CODE,
      );
```

**Comment by gbinal** [comment]
Note 12-21-23 email to team

**Comment by gbinal** [comment]
@akuny - as a next move, can you get a sense of level of effort for adding in ^^^^.

**Comment by akuny** [comment]
Here's a breakdown of what the tool we're using for a11y scans can do for those WCAG criteria above: https://docs.google.com/document/d/1I4Tlj-M5vD-eP6eX-bAFRI9ZUOCDJ_-3GHVt8JBsHSU/edit?usp=sharing

**Comment by gbinal** [comment]
this is done and can be closed in favor of #741
